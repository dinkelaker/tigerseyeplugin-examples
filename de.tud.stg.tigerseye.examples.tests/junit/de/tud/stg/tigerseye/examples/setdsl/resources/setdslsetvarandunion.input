Set a = { "2"} â‹ƒ { "6", "8", "2"}; 
        
Set ares = a â‹ƒ { "1" , "2", "3" }