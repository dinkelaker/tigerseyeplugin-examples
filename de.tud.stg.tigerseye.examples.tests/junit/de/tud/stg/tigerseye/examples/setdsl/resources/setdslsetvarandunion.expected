Set a = union(
asSet(
singleElementedList(
"2")),
asSet(
multiElementedList(
"6",
multiElementedList(
"8",
singleElementedList(
"2"))))); 
        
    Set ares = union(
a,
asSet(
multiElementedList(
"1",
multiElementedList(
"2",
singleElementedList(
"3")))))